modules/programs/direnv.nix

```
@@ -24,6 +24,13 @@ in {
   options.programs.direnv = {
     enable = mkEnableOption "direnv, the environment switcher";
 
+    package = mkOption {
+      description = "direnv derivation to use.";
+      type = types.package;
+      default = pkgs.direnv;
+      defaultText = literalExpression "pkgs.direnv";
+    };
```

Can use `mkPackageOption` here 🙂

Something like

```suggestion
    package = mkPackageOption pkgs "direnv" { };
```

---

Same for nix-direnv…

---

Cool, I didn't know about that function, done!